server/controllers/users.controller.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const { UsersRepository } = require('../ repositories');
+
+class UsersController {
+  constructor() {}
+
+  async getAllUsers(req, res) {
+    const data = await UsersRepository.findAll();
+    return res.status(200).send(data.rows);
+  }
+
+  async getUserById(req, res) {
+    const id = req.params.id;
+    const data = await UsersRepository.findById(id);
```

Comment:
I think there should be a check for if id is undefined.

Reply:
Got it, But if id  undefined, will it route in this function?